src/home/index.js

```diff
@@ -0,0 +1,12 @@
+import makeUser from './make-user.js';
+import api from '../services/api.js';
+
+const userSignUp = document.getElementById('form');
+
+userSignUp.addEventListener('submit', (event) => {
```

Comment:
using arrow functions with one parameter you can use the current syntax you are using. An alternative is if there is only one parameter you can write something like 
```userSignUp.addEventListener('submit', event => {
}
```
without the need for the parenthesis.